### PR TITLE
Adding Aozora (hotswap) keyboard 

### PR DESCRIPTION
**Pull request by Phooood**
# Description
Adding firmware for hotswap version of [Aozora](https://salmoncat.studio/pages/aozora-%E9%9D%92%E7%A9%BA-6-9-21-7-10-21)
# Types of Changes
- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [x] Keyboard (addition or update)
- [x] Keymap/layout/userspace (addition or update)
- [ ] Documentation

# Issues Fixed or Closed by this PR

# Checklist
- [x] My code follows the style of this project: [C](https://docs.qmk.fm/#/coding_conventions_c), [Python](https://docs.qmk.fm/#/coding_conventions_python)
- [x] I have read the [**PR Checklist** document](https://beta.docs.qmk.fm/developing-qmk/pr_checklist) and have made the appropriate changes.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage)

### CODE REVIEW

**Review by drashna — APPROVED** [review]

File: keyboards/aozora/hotswap/keymaps/default/readme.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
+# Aozora
```
Comment (fauxpark) [inline keyboards/aozora/hotswap/keymaps/default/readme.md:1]: This file can be deletd.

File: keyboards/aozora/hotswap/readme.md [hunk 2]
```diff
@@ -0,0 +1,17 @@
+# Aozora 
+
+A 65%-Style 6-Key keyboard featuring a cloud-shaped badge. 
+
+![SCS Aozora](https://i.imgur.com/dL5nonc.jpg)
+
+Keyboard Maintainer: [Food](https://github.com/Phooood)
+Hardware Supported: [Aozora](https://salmoncat.studio/pages/aozora-%E9%9D%92%E7%A9%BA-6-9-21-7-10-21)
+Hardware Availability: [salmoncat.studio](https://salmoncat.studio/pages/aozora-%E9%9D%92%E7%A9%BA-6-9-21-7-10-21) | GB is over.
```
Comment (fauxpark) [inline keyboards/aozora/hotswap/readme.md:9]:
```suggestion
* Keyboard Maintainer: [Food](https://github.com/Phooood)
* Hardware Supported: [Aozora](https://salmoncat.studio/pages/aozora-%E9%9D%92%E7%A9%BA-6-9-21-7-10-21)
* Hardware Availability: [salmoncat.studio](https://salmoncat.studio/pages/aozora-%E9%9D%92%E7%A9%BA-6-9-21-7-10-21) | GB is over.
```

File: keyboards/aozora/readme.md [hunk 3]
```diff
@@ -0,0 +1,17 @@
+# Aozora
```
Comment (fauxpark) [inline keyboards/aozora/readme.md:1]: This file can be deleted.

**Review by fauxpark — CHANGES_REQUESTED** [review]

**Review by fauxpark — APPROVED** [review]

**Comment by drashna** [comment]
Thanks!

**Pull request merged**